### DISCUSSION TITLE
Teleporting

### DISCUSSION THREAD

**Discussion by thatdumbdevit** [FabricMC/fabric-api#2195] [Mod Dev Support]
Hi,
I'm trying to make a block that works like a portal, without using the [Kyrptonaught library](https://github.com/kyrptonaught/customportalapi), as suggested in the [Fabric Wiki](https://fabricmc.net/wiki/tutorial:custom_portals), because of compatibility problems. How can I do it?

I was thinking about Mixing, but also using the command
`execute in <dimension name> run tp @p <coords>`

**Comment by warjort** [comment]
https://github.com/FabricMC/fabric/blob/1.18.2/fabric-dimensions-v1/src/main/java/net/fabricmc/fabric/api/dimension/v1/FabricDimensions.java

Here's an example using it:
https://github.com/modmuss50/SimpleVoidWorld/blob/1.18/src/main/java/me/modmuss50/svw/BlockPortal.java
